Form-Controls/styles.css

```diff
@@ -0,0 +1,145 @@
+/* Custom properties */
+:root {
+    --primary-100: #FF7F50;
+    --primary-200: #dd6236;
+    --primary-300: #8f1e00;
+    --accent-100: #8B4513;
+    --accent-200: #ffd299;
+    --text-100: #000000;
+    --text-200: #2c2c2c;
+    --bg-100: #F7EEDD;
+    --bg-200: #ede4d3;
+    --bg-300: #c4bcab;
+}
+
+/* Reset and base styles */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Segoe UI', system-ui, sans-serif;
+}
+
+body {
+    background-color: var(--bg-100);
+    color: var(--text-200);
+    line-height: 1.6;
+}
+
+/* Header styles */
+header {
+    background-color: var(--primary-300);
+    color: var(--bg-100);
+    padding: 2rem 0;
+    text-align: center;
+    margin-bottom: 2rem;
+    box-shadow: 0 2px 4px rgba(143, 30, 0, 0.2);
+}
+
+header h1 {
+    font-size: 2.5rem;
+    font-weight: 300;
+}
+
+/* Main content and form styles */
+main {
+    max-width: 600px;
+    margin: 0 auto;
+    padding: 0 1rem;
+}
+
+form {
+    background-color: white;
+    padding: 2rem;
+    border-radius: 10px;
+    box-shadow: 0 4px 6px rgba(139, 69, 19, 0.15);
+    border: 1px solid var(--bg-200);
+}
+
+form div {
+    margin-bottom: 1.5rem;
+}
+
+label {
+    display: block;
+    margin-bottom: 0.5rem;
+    color: var(--accent-100);
+    font-weight: 500;
+}
+
+label span {
+    color: var(--primary-200);
+}
+
+input, select {
+    width: 100%;
+    padding: 0.8rem;
+    border: 2px solid var(--bg-200);
+    border-radius: 6px;
+    font-size: 1rem;
+    transition: all 0.3s ease;
+    background-color: var(--bg-100);
+    color: var(--text-100);
+}
+
+input:focus, select:focus {
```

Comment:
Could you check your focus that it's working or not?

Reply:
I tested this, I expected when I clicked the form area it lit up the input area and it worked.
What did you expect instead?

Reply:
Have you tried finding out which HTML elements can receive focus? Maybe you missed some elements?